anchor-service/src/cas_mock.rs

```diff
@@ -75,7 +75,7 @@ impl Store for MockAnchorEventService {
         highwater: i64,
         limit: i64,
     ) -> Result<Vec<AnchorRequest>> {
-        let ints = ((highwater as u64)..self.anchor_req_count).take(limit as usize);
+        let ints = ((highwater as u64 + 1)..=self.anchor_req_count).take(limit as usize);
```

Comment:
this is why all the test vectors channged

Reply:
Can you add a comment why we are adding 1 to the water mark?